tests/func/remote/test_http.py

```diff
@@ -0,0 +1,12 @@
+from dvc.main import main
+
+
+def test_incompatibility_with_no_traverse(dvc_repo, caplog):
```

Comment:
Just create a unit test with initializing RemoteHTTP in tests/unit/remote/test_http.py